python/API_Examples_Python.py

```diff
@@ -0,0 +1,200 @@
+    pay_load = json.dumps(pay_load)
+    r = requests.post(mainnet_url + "/v1/authenticate", pay_load)
+    if r.status_code == requests.codes.ok:
+        resp = r.json()
+        user_information["token"] = resp['token']
+        print("Login Successful")
+    else:
+        print("ERROR: User login post did not work. ")
+
+def get_balance():
+    """
+    Get the balances of all tokens of a trader.
+    """
+    headers = {"Authorization": user_information["token"]}
+    resp = requests.get(mainnet_url + "/v1/balances/" + user_information["traderAddr"], headers=headers)
+    if resp.status_code == requests.codes.ok:
+        pprint.pprint(resp.json())
+    else:
+        print("ERROR: Get Balance request did not work. ")
+
+def place_order():
+    """
+    Place a new order
+    """
+    headers = {"Authorization": user_information["token"]}
+    signature = example_sign_order.play()
+    #print("Signature: ", signature)
+    post_data = {'pairId': 'ETH_ZIL',
+                 'action': 'Buy',
+                 'amount': '660.00',
+                 'price': '0.0001600',
+                 'nonce': 3640819234038,
+                 'expireTimeSec': 1531251449,
+                 'traderAddr': user_information["traderAddr"],
+                 'sig': signature}
+    post_data = json.dumps(post_data)
+    order_response = requests.post(mainnet_url + "/v1/placeorder", post_data, headers=headers)
+    if order_response.status_code == requests.codes.ok:
+        pprint.pprint(order_response.json())
+    else:
+        print(order_response.content)
+        print("ERROR: Place Order post did not work. ")
+
+def cancel_order():
+    """
+    cancel a single specific order
+    """
+    headers = {"Authorization": user_information["token"]}
+    pay_load = {"traderAddr": user_information["traderAddr"],
+                "orderId": "10827893",
+                "pairId": "ETH_ZIL",
+                "nonce": "2640819234038"}
+    pay_load = json.dumps(pay_load)
+    cancel_order_response = requests.post(mainnet_url + "/v1/cancelorder", pay_load, headers=headers)
+    if cancel_order_response.status_code == requests.codes.ok:
+        print("ok")
+    else:
+        print("ERROR: Cancel Order post did not work. ")
+
+def cancel_all_orders():
+    """
+    # Cancel all existing orders using given parameters
+    """
+    headers = {"Authorization": user_information["token"]}
+    pay_load = {"traderAddr": user_information["traderAddr"],
+                "pairId": 'ETH_ZIL',
+                "nonce": '3640819234038'}
+    pay_load = json.dumps(pay_load)
+    response = requests.post(mainnet_url + "/v1/cancelallorders", pay_load, headers=headers)
+    print(response.content)
+
+def get_active_orders():
+    """
+    Get unfilled or partially filled orders that have not been cancelled or expired of a trader.
+    """
+    headers = {"Authorization": user_information["token"]}
+    pay_load = {"pairId": 'ETH_ZIL',
```

Comment:
No need of payload, like `get_trade_history`. The same for rest GET request.

Reply:
![image](https://user-images.githubusercontent.com/30614865/42405443-89ab434a-814a-11e8-9228-e39eb670870e.png)